modules/ROOT/pages/configure-apq-cdn.adoc

```diff
@@ -0,0 +1,150 @@
+
+== Register an Automatic Persisted Query to DataGraph
+
+To post a persisted query ensure you have:
+
+* The DataGraph Consume permission
+* An authenticated connection to DataGraph using a valid client ID and secret
+* Created a ID for the persisted query
+
+To register a persisted query, create a POST request that contains the following:
+
+* The endpoint for the DataGraph instance
+* An extension that contains the `persistedQuery` parameter
+* An ID within the `persistedQuery` parameter
+* A GraphQL query and any variables 
+
+The following example shows how to register a persisted query using cURL:
+
+----
+curl --location --request POST 'https://<datagraph-endpoint>/graphql' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "extensions": {
+        "persistedQuery": {
+            "version": 1,
+            "sha256Hash": "bcd3868f0cb4f723cdb5bdbc433d6ab73156f0b9d0e9d5bd17e200cdab6fba4"
+        }
+    },
+    "query": "{\n    customers(customersCount: 30) {\n        customerId\n        name\n    }\n}",
+    "variables": {}
+}
+----
+
+== Request a Persisted Query from DataGraph
+
+To request a persisted query from DataGraph, ensure you have:
+
+* The DataGraph Consume permission
+* An authenticated connection to DataGraph using a valid client ID and secret
+* 
+
+To request a persisted query, create a GET request that contains the following:
+* The endpoint for the DataGraph instance
+* An extension that contains the `persistedQuery` parameter
+* An ID within the `persistedQuery` parameter
+
+The following example shows how to request a persisted query using cURL:
+
+----
+curl --location --request GET 'https://<datagraph-endpoint>/graphql?extensions=%7B%0A%20%20%22persistedQuery%22%3A%20%7B%0A%20%20%20%20%22version%22%3A%201%2C%0A%20%20%20%20%22sha256Hash%22%3A%20%22acf31818e50ac3e818ca4bdbc433d6ab73176f0b9d5f9d5ad17e200cdab6fba4%22%0A%20%20%7D%0A%7D'
+----
+
+== Troubleshoot Query Not Found Errors
+
+If you send an ID in a request and a query is not found, DataGraph returns an error that the query is not persisted.
+
+----
+{
+   "errors": [
+       {
+           "message": "PersistedQueryNotFound",
+           "locations": [],
+           "extensions": {
+               "persistedQueryId": "bcd3868f0cb4f723cdb5bdbc433d6ab73156f0b9d0e9d5bd17e200cdab6fba4",
+               "classification": "PersistedQueryNotFound"
+           }
+       }
+   ]
+}
+----
+
+At this point, you can send a new POST request with a query and ID to register the query and can then subsequently request the query. 
+
+== Using Automatic Persisted Queries with a CDN
+
+Content Delivery Networks (CDNs) enable you to cache full responses to GET operations to speed performance of your APIs. 
+
+Since the cache functionality of many CDNs only support GET requests, you can sync persisted queries in DataGraph to your CDN for fast retrieval. 
```

Comment:
localization
```suggestion
Because the cache functionality of many CDNs supports only GET requests, you can sync persisted queries in DataGraph to your CDN for fast retrieval. 
```